deploy/02-sa.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
Creating a separate service account for `console-operator` and updating bindings below to use it.